generators/app/templates/service/src/utils/query.ts

```diff
@@ -1,15 +1,15 @@
 import { QueryOptions } from '@cxcloud/ct-types/common';
 
 export function getQueryOptions(
-    page?: number,
-    perPage?: number,
-    sortPath?: string,
-    ascending?: boolean
+	page?: number,
+	perPage?: number,
+	sortPath?: string,
+	ascending?: boolean
 ): QueryOptions {
-    return {
-        page: typeof page !== 'number' || page < 1 ? 1 : page,
-        perPage: typeof perPage !== 'number' || perPage < 1 ? 20 : perPage,
-        sortPath,
-        ascending
-    };
-}
\ No newline at end of file
+	return {
+		page: typeof page !== 'number' || page < 1 ? 1 : page,
```

Comment:
The indentation here is still wrong. You are using tabs instead of spaces. Like I said Prettier can help you with this